akka-javasdk/src/main/java/akka/javasdk/agent/impl/PersistentCoreMemory.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.javasdk.agent.impl;
```

Comment:
shouldn't this be in `akka.javasdk.impl.agent` package?

Reply:
most likely yes, good call 👍